components/controls/ClipboardButton.tsx

```diff
@@ -65,25 +66,41 @@ export default class ClipboardButton extends React.PureComponent<Props, State> {
     }
   }
 
-  showTooltip = () => {
+  handleMouseEnter = () => {
     if (this.mounted) {
       this.setState({ tooltipShown: true });
+    }
+  };
+
+  handleMouseLeave = () => {
+    if (this.mounted) {
+      this.setState({ tooltipShown: false });
+    }
+  };
+
+  showTooltip = () => {
+    if (this.mounted) {
+      this.setState({ isCopied: true, tooltipShown: true });
       setTimeout(() => {
         if (this.mounted) {
-          this.setState({ tooltipShown: false });
+          this.setState({ isCopied: false, tooltipShown: false });
         }
       }, 1000);
     }
   };
 
   render() {
     return (
-      <Tooltip overlay={translate('copied_action')} visible={this.state.tooltipShown}>
+      <Tooltip
+        onMouseEnter={this.handleMouseEnter}
+        onMouseLeave={this.handleMouseLeave}
+        overlay={translate(this.state.isCopied ? 'copied_action' : 'copy_to_clipboard')}
+        visible={this.state.tooltipShown}>
         <Button
           className={classNames('no-select', this.props.className)}
           data-clipboard-text={this.props.copyValue}
           innerRef={node => (this.copyButton = node)}>
-          {this.props.label ? this.props.label : translate('copy')}
+          {this.props.children || (this.props.label ? this.props.label : translate('copy'))}
```

Comment:
Let's completely drop label and always use children instead. And if there are no children prop we display the `translate('copy')`.